specification.md

```diff
@@ -2855,6 +2859,65 @@ the needed events at the defined times to trigger workflow instance creation.
 
 The `directInvoke` property defines if workflow instances are allowed to be created outside of the defined interval or cron expression.
 
+#### Cron Definition
+
+| Parameter | Description | Type | Required |
+| --- | --- | --- | --- |
+| expression | Repeating interval (cron expression) describing when the workflow instance should be created | string | yes |
+| deadline | Specific date and time (ISO 8601 format) when the cron expression invocation should be stopped | string | no |
```

Comment:
I'm not sure I've experienced a need for this deadline use-case before but it makes me wonder if we should also add a corresponding attribute for 'start time' or 'valid-after'?

Reply:
@jorgenj idk what do you think? To me a cron definition does in a way define the start time (next time the cron fires after deployment). validAfter could be useful. @ricardozanini wdyt too?

Reply:
I think there are very limited use-cases that I've seen where a 'validAfter' would've been useful but there were always easy alternatives, but since we're adding an 'end' it seemed relevant to consider adding a 'begin' as well.   My vote is probably to not add 'validAfter' unless someone actually asks for it.